tests/test_plugin_kraken_csv_download.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2023 Neal Chambers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# pylint: disable=protected-access
+
+from datetime import datetime
+from os import listdir, makedirs, path, remove, unlink
+from typing import Any, List, Optional
+
+from rp2.rp2_decimal import RP2Decimal
+from rp2.rp2_error import RP2RuntimeError
+
+from dali.cache import CACHE_DIR
+from dali.historical_bar import HistoricalBar
+from dali.plugin.pair_converter.csv.kraken import Kraken
+
+_CACHE_DIRECTORY: str = "input/kraken_test"
+
+
+class TestKrakenCsvDownload:
+    def test_chunking(self, mocker: Any) -> None:
+        kraken_csv = Kraken("whatever")
+
+        # Flush test cache directory
+        for filename in listdir(_CACHE_DIRECTORY):
+            file_path = path.join(_CACHE_DIRECTORY, filename)
+            try:
+                if path.isfile(file_path):
+                    unlink(file_path)
+            except RP2RuntimeError:
+                pass
+
+        cache_path = path.join(CACHE_DIR, kraken_csv.cache_key())
+        if path.exists(cache_path):
+            remove(cache_path)
+
+        mocker.patch.object(kraken_csv, "_Kraken__CACHE_DIRECTORY", _CACHE_DIRECTORY)
+        if not path.exists(_CACHE_DIRECTORY):
+            makedirs(_CACHE_DIRECTORY)
+        with open("input/USD_OHLCVT_test.zip", "rb") as file:
```

Comment:
Just wanted to make sure that input/USD_OHLCVT_test.zip has made-up data in it and not pasted data from their files: if we're going to ship it with our distro it has to contain fake data we use for testing.

Reply:
oh curses, yeah, stupid licensing. I'll edit the amounts to be completely made up.